src/mcpwm.rs

```diff
@@ -122,25 +122,171 @@ impl From<CounterMode> for mcpwm_counter_type_t {
+    ///               |   .                |
+    /// ---------------   .                ---------------------
+    ///               .   .                .
     BypassFallingEdge { red: u16 },
 
     /// MCPWM_ACTIVE_HIGH_MODE
+    ///
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .--------------------.   .
+    ///               |   .                |   .
+    /// MCPWMXA in    |   .                |   .
+    ///               |   .                |   .
+    /// ---------------   .                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .   .----------------.   .
+    ///          red >.   |<               |   .
+    /// MCPWMXA out   .   |                |   .
+    ///               .   |                |   .
+    /// -------------------                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .------------------------.
+    ///               |   .               >.   |< fed
+    /// MCPWMXB out   |   .                .   |
+    ///               |   .                .   |
+    /// --------------.   .                .   -----------------
+    ///               .   .                .   .
     ActiveHigh { red: u16, fed: u16 },
 
     /// MCPWM_ACTIVE_LOW_MODE
+    ///
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .--------------------.   .
+    ///               |   .                |   .
+    /// MCPWMXA in    |   .                |   .
+    ///               |   .                |   .
+    /// ---------------   .                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    /// ------------------.                .--------------------
+    ///          red >.   |<               |   .
+    /// MCPWMXA out   .   |                |   .
+    ///               .   |                |   .
+    ///               .   ------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    /// --------------.   .                .   .----------------
+    ///               |   .               >.   |< fed
+    /// MCPWMXB out   |   .                .   |
+    ///               |   .                .   |
+    ///               --------------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
     ActiveLow { red: u16, fed: u16 },
 
-    /// MCPWM_ACTIVE_HIGH_COMPLIMENT_MODE
+    // TODO: Is this actually true? --------
+    //                                      |
+    //                                      v
+    /// MCPWM_ACTIVE_HIGH_COMPLIMENT_MODE - The most common deadtime mode
```

Comment:
> The most common deadtime mode
This mode seemed like the one making most sense to me... Is this true or should I remove that comment? :)

Reply:
It might be true (idk :D) but I'd say remove the comment regardless.